Makes second set of changes to IPI docs (#113832)
openshift/openshift-docs: modules/installation-azure-tested-machine-types.adoc

[id="installation-azure-tested-machine-types_{context}"]
= Tested instance types for Azure

[role="_abstract"]

> You can use this reference to find Microsoft Azure instance types tested with {product-title}. Choose a listed instance type when you install a cluster on 64-bit x86 infrastructure.

Reviewer: Similar comment with similar suggestion, up to you:

```suggestion
There are several Microsoft Azure instance types tested with {product-title}. Choose a listed instance type when you install a cluster on 64-bit x86 infrastructure.
```
